From 021b88dcdc646d1b2d5afe438ec5a87ff0980704 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:37:37 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(8afedbb)..lineage-16.0-android-9.0.0_r31(d26dbc9) (DO NOT
 SUBMIT)

Andrew Chant (1):
 - bootcontrol: Add TARGET_USES_HARDWARE_QCOM_BOOTCTRL

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5044688 from f5db01c3b14d720f3d603cfb3b887d89c2b11b28 to
   pi-qpr2-release

Change-Id: I7022241ac45be09a70df71084ef4d1738b60ca08

Change-Id: Iaf72d848684145cd4c9fd3da54ca59b24f3673b3
---
 Android.mk | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/Android.mk b/Android.mk
index 51648f4..0e74382 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,7 +1,11 @@
-ifeq ($(AB_OTA_UPDATER),true)
+# Preset TARGET_USES_HARDWARE_QCOM_BOOTCTRL for existing platforms.
+ifneq ($(filter msm8953 msm8996 msm8998 sdm660 sdm845 sdm710,$(TARGET_BOARD_PLATFORM)),)
+TARGET_USES_HARDWARE_QCOM_BOOTCTRL := $(AB_OTA_UPDATER)
+endif
+
+ifeq ($(strip $(TARGET_USES_HARDWARE_QCOM_BOOTCTRL)),true)
 # TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
 ifneq ($(BOARD_IS_AUTOMOTIVE),true)
-ifneq ($(filter msm8953 msm8996 msm8998 sdm660 sdm845 sdm710,$(TARGET_BOARD_PLATFORM)),)
 LOCAL_PATH := $(call my-dir)
 
 # HAL Shared library for the target. Used by libhardware.
@@ -29,4 +33,3 @@ include $(BUILD_STATIC_LIBRARY)
 
 endif
 endif
-endif
-- 
2.17.1

